### DISCUSSION TITLE
Fail to deploy in Github Pages

### DISCUSSION THREAD

**Discussion by joshbushdash** [byanko55/jekyll-theme-satellite#42] [Q&A] [answered]
Hi, I'm new to github pages. This templete is rendering well locally, However, it will not be deployed sucessfully on Github Page.
It sucessed with source file ./(root), but failed with source file /docs, the error is as follows.

<img width="1213" alt="Screenshot 2024-04-28 at 19 39 39" src="https://github.com/byanko55/jekyll-theme-satellite/assets/35750911/05f09bcc-7776-42e5-b538-686fda5d885f">

No matter in either way, the rendering of the page failed, like this, https://joshbushdash.github.io/test.github.io/. (However, I have checked the deployment of the original source file, https://byanko55.github.io/jekyll-theme-satellite/, it successed.)

<img width="1641" alt="Screenshot 2024-04-28 at 19 14 36" src="https://github.com/byanko55/jekyll-theme-satellite/assets/35750911/546d2aec-0916-4007-91f7-bc53ffcaf367">

Thank you for your help!

**Comment by byanko55** [comment, marked as answer]
**The repository should be EXACTLY username.github.io** (_zihao-zheng.github.io → joshbushdash.github.io_).

Otherwise, the root URL of your git page will be changed, which makes the website fail to load page resources (e.g., image, css, javascripts).

Please read this [article](https://docs.github.com/en/pages/quickstart) if you haven't. 

![image](https://github.com/byanko55/jekyll-theme-satellite/assets/132314568/f00bb13c-c020-464e-be06-9c3e59a6f2f5)

**Reply by joshbushdash** [reply]
Problem resolved!

Thank you very much~
